lib/individual/user_chats_menu.dart

```diff
@@ -3,7 +3,7 @@ import 'package:cura/shared/widgets/navigation-bar.dart';
 import 'package:flutter/material.dart';
 import 'package:cura/individual/home_page_individual.dart';
 import 'package:flutter_screenutil/flutter_screenutil.dart';
-import '../shared/widgets/gradient_background.dart';
+import '../shared/widgets/gradient_background.dart' as shared;
```

Comment:
Since `UniDirectionalGradient()` is present only in one location now due to your changes, i.e. in `shared`, kindly remove the `as shared` construct, as this leads to unintentional verbosity.

Reply:
Fixed in commit e0b6cc4